core/src/main/java/hudson/model/StringParameterDefinition.java

```diff
@@ -69,7 +70,7 @@ public StringParameterValue getDefaultParameterValue() {
         return v;
     }
 
-    @Extension
+    @Extension @Symbol({"string","stringParam"})
```

Comment:
Note that you really need to do `StringParameterValue` too for Pipeline support, though this could be subsumed under the proposed change mentioned in [JENKINS-26093](https://issues.jenkins-ci.org/browse/JENKINS-26093): making that an `Extension` too. [reference](https://github.com/jenkinsci/symbol-plugin/blob/66fdafc0ca3bf733819819479f75fb05f22ae0dc/plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java#L374-L388)